src/test/java/org/cactoos/list/ImmutableTest.java

```diff
@@ -45,15 +45,14 @@
 public class ImmutableTest {
 
     @Test
-    public void innerListMustNotBeChanged() {
+    public void innerListIsDecorated() {
         final List<String> strings = new ArrayList<>(Arrays.asList("a", "b", "c"));
         final List<String> immutable = new Immutable<>(strings);
-        final int original = immutable.size();
         strings.add("d");
         new Assertion<>(
-            "inner list must not be changed",
-            original,
-            new IsEqual<>(immutable.size())
+            "must reflect inner list in the decorator",
```

Comment:
@victornoel Please put the first letter in uppercase in assertion messages. Also, I think that we should compare the elements of the iterators and not just their size